x-pack/plugins/features/server/oss_features.test.ts

```diff
@@ -50,6 +50,7 @@ Array [
       const privileges = [];
       for (const featurePrivilege of featurePrivilegeIterator(new KibanaFeature(featureConfig), {
         augmentWithSubFeaturePrivileges: true,
+        licenseType: 'enterprise',
```

Comment:
Warning: this comment may actually be complete and utter nonsense.

Changing this to `licenseType: 'basic'`, the tests still pass. It looks like this is expected as we don't have any OSS features with minimum-licensed sub-feature privileges. However, this may change in the future. Perhaps we should add another set of test cases with the basic license to be safe?

Reply:
You're right that we don't have any OSS features with licensed minimum sub-feature privileges yet. We will though if/when reporting takes advantage of this, though.

The addition of `licenseType` in this test was solely to satisfy the new function signature.

> Perhaps we should add another set of test cases with the basic license to be safe?

Yeah I think that's worthwhile